sdk/helper/pluginutil/runner.go

```diff
@@ -86,3 +96,9 @@ func CtxCancelIfCanceled(f context.CancelFunc, ctxCanceler context.Context) chan
 	}()
 	return quitCh
 }
+
+// MultiplexingSupport returns true if a plugin supports multiplexing.
+// Currently this is hardcoded for database plugins.
+func MultiplexingSupport(version int) bool {
+	return version == 6
+}
```

Comment:
Double check me here, but I believe this isn't called anywhere now. Could probably be removed

Reply:
Yep, I meant to remove this. Thanks